v2/thirdpartyemailpassword/common-customizations/embed-sign-in-up-form.mdx

```diff
@@ -125,7 +321,29 @@ import Header from "./header";
 import Footer from "./footer";
 import { useHistory } from "react-router-dom5";
 
-function MyLandingPage() {
+SuperTokens.init({
+    appInfo: {
+        apiDomain: "...",
+        appName: "...",
+        websiteDomain: "..."
+    },
+    recipeList: [
+        ThirdPartyEmailPassword.init({
+            // ...
+        }),
+        // ...
+    ],
+    // highlight-start
+    async getRedirectionURL(context) {
+        // The user will be taken to this path when they need to login.
+        if (context.action === "TO_AUTH") {
+            return "/auth"; // return the path where you are rendering the Auth UI
+        };
+    },
+    // highlight-end
+});
+
+function LandingPage() {
```

Comment:
the whole LandingPage component needs to be highlighted, and you should have some text below explaining why we are rendering `<Session.SessionAuth>` if a session exists

Reply:
done